src/github.com/getlantern/launcher/launcher_darwin.go

```diff
@@ -44,10 +47,17 @@ func CreateLaunchFile(autoLaunch bool) {
 	var content bytes.Buffer
 	fname := appdir.InHomeDir(LaunchdPlistFile)
 
+	lanternPath, err := osext.Executable()
+	if err != nil {
+		log.Errorf("Could not get Lantern directory path: %q", err)
+		return
+	}
+	log.Debugf("Using lantern path: %v", lanternPath)
+
 	// Create plist template and set RunAtLoad property
 	t := template.Must(template.New("LaunchdPlist").Parse(LaunchdPlist))
 
-	err = t.Execute(&content, &Plist{RunAtLoad: autoLaunch})
+	err = t.Execute(&content, &Plist{RunAtLoad: autoLaunch, Path: lanternPath})
```

Comment:
This also actually fixes the bug where users putting Lantern in a directory other than /Applications/Lantern.app would have issues with auto-start.